mobile/src/libs/components/header/header.tsx

```diff
@@ -35,41 +31,44 @@ const Header: React.FC<Properties> = ({
 
   const hasValue = Boolean(badgeCount);
 
-  const handleGoBack = (): void => {
-    navigation.goBack();
-  };
-
   const handleIconPress = (): void => {
     navigation.navigate(ProfileScreenName.PROFILE);
   };
 
   return (
-    <View
-      style={[
-        styles.header,
-        isArrowVisible && styles.headerCenter,
-        isProfileVisible && styles.settings,
-      ]}
-    >
-      {isArrowVisible && (
-        <TouchableOpacity style={styles.arrow} onPress={handleGoBack}>
-          <Icon name="back-arrow" color={AppColor.BLUE_200} />
-        </TouchableOpacity>
-      )}
-      <View style={styles.titleBadgeContainer}>
-        <Text style={styles.title}>{title ?? name}</Text>
-        {hasValue && <Badge count={badgeCount} />}
-      </View>
-      {isProfileVisible && (
-        <View style={styles.settingsContainer}>
-          <IconButton
-            onPress={handleIconPress}
-            color={AppColor.BLUE_300}
-            iconName="user"
-          />
+    <>
```

Comment:
```suggestion
```

Reply:
can't do it 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/108797614/326f8063-019e-4692-91ed-baee45adb8cf)

Reply:
don't you return one element? if so this should work
```
if (isVisible) {
  return <View>{children}</View>
}

return <View>{children}</View>
```